### PR TITLE
Session-02 Assignments

### PR DESCRIPTION
**Pull request by burrows99**

### CODE REVIEW

**Comment by imranmd** [comment]
Can you add github pages url here once you deploy? [Not urgent though]

File: Week-1/Assignments/Session-02/Intermediate/Hometown webpage.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (imranmd) [inline Week-1/Assignments/Session-02/Intermediate/Hometown webpage.html:2]: nit: Hope we have understanding of lang attribute here.

File: Week-1/Assignments/Session-02/Advanced/NewsArticle-Webpage/NewsArticle-Webpage.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>5 Things to Know About Rattlesnakes and Their Babies</title>
```
Comment (imranmd) [inline Week-1/Assignments/Session-02/Advanced/NewsArticle-Webpage/NewsArticle-Webpage.html:5]: Good work

File: Week-1/Assignments/Session-02/Intermediate/Poem- Webpage.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <br> 
+        Debitis, laboriosam necessitatibus amet cumque labore 
+        <br>
+        earum doloribus suscipit incidunt quisquam dolor
+        <br>
+        odit pariatur quidem nihil sit molestias ab alias magnam aliquid.
+    </p>
+    <p>
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+        <br>
```
Comment (imranmd) [inline Week-1/Assignments/Session-02/Intermediate/Poem- Webpage.html:21]: Good work here. using too many break statements is good practice? Or too many DOM elements? Are there any other alternatives?

File: Week-1/Assignments/Session-02/Intermediate/list.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (imranmd) [inline Week-1/Assignments/Session-02/Intermediate/list.html:7]: Meaningful title and descriptions

File: Week-1/Assignments/Session-02/Intermediate/table.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (imranmd) [inline Week-1/Assignments/Session-02/Intermediate/table.html:7]: Good work except the title

**Review by imranmd — APPROVED** [review]
Approving with suggestions

File: Week-1/Assignments/Session-02/Intermediate/Hometown webpage.html (hunk 1 above)
Comment (burrows99) [reply Week-1/Assignments/Session-02/Intermediate/Hometown webpage.html:2]: yes. to specify language of content

File: Week-1/Assignments/Session-02/Intermediate/Poem- Webpage.html (hunk 3 above)
Comment (burrows99) [reply Week-1/Assignments/Session-02/Intermediate/Poem- Webpage.html:21]: using li with ul using style of  list-style-type: none

**Pull request merged**